include/vkgcDefs.h

```diff
@@ -405,6 +406,10 @@ struct PipelineOptions {
   bool reserved1f;                                       /// Reserved for future functionality
   bool enableInterpModePatch; ///< If set, per-sample interpolation for nonperspective and smooth input is enabled
   bool pageMigrationEnabled;  ///< If set, page migration is enabled
+#if LLPC_CLIENT_INTERFACE_MAJOR_VERSION > 52
```

Comment:
I would prefer to use >= 53 instead of > 52, to be consistent with (the small number of) other uses, and so it reads as "this is new on version 53". Being consistent might help in the future with removing now-unnecessary #ifs when the client driver is known to have stepped past that point.

Same for other places this is used.

Reply:
I believe I have fixed all of the location.